### PR TITLE
render enum in table

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

**Comment by greyli** [comment]
Please add some tests for this change and update the changelog.

**Comment by PanderMusubi** [comment]
Tests are already in https://github.com/helloflask/bootstrap-flask/pull/271/commits/3e5507d1b9773231ee7af0dd5bcfc8ab668ae7ac#diff-5041814f41c9cf2e44c8ee9ec3f448747d7f32492b0d0586f4eda9b7f69a6b27

Update of changes is in https://github.com/helloflask/bootstrap-flask/pull/271/commits/b8f87c9150c7720b364dc5930a639d0ab9fdf837

**Comment by PanderMusubi** [comment]
@greyli can you review this? Thanks.

File: CHANGES.rst [hunk 1]
```diff
@@ -7,6 +7,8 @@ Changelog
 
 Release date: -
 
+- Renden enums in tables by their labels.
```
Comment (greyli) [inline CHANGES.rst:10]:
```suggestion
- Render enums in tables by their labels.
```

**Review by greyli — CHANGES_REQUESTED** [review]

**Pull request merged**